README.md

```diff
@@ -120,6 +120,7 @@ needs_to_merge_symbol              | `ᄉ`
 needs_to_merge_color               | `$yellow`
 has_upstream_symbol                | `⇅`
 has_upstream_color                 | `$on`
+has_no_upstream_symbol             | ` ⃢`
```

Comment:
I don't know this symbol, it probably comes from a patched font, however I'd rather have a standard unicode symbol as default. Because not everyone may have this symbol. Also the space may not be necessary.

Reply:
The symbol is the unicode character [SCREEN](http://www.fileformat.info/info/unicode/char/1f5b5/index.htm)

![no-upstream_unicode-screen](https://user-images.githubusercontent.com/4189991/37619578-2cbdc396-2bba-11e8-896a-24dcc2633142.png)

However, it does not seem to be supported by many fonts. I replaced it with a character from your code.

![no-upstream_unicode](https://user-images.githubusercontent.com/4189991/37619669-707ff932-2bba-11e8-99ef-23921f608320.png)

An alternative would be to leave it empty and it can then be adjusted by the user via the variable.

I use a font from 'Awesome-Terminal-Fonts' with the symbol 'omg_not_tracked_branch_symbol' from 'oh-my-git'.
![no-upstream_awesome_omg-not-tracked](https://user-images.githubusercontent.com/4189991/37619682-78afa986-2bba-11e8-88ce-a9be1af14d12.png)